backend/tests/environments/test_router.py

```diff
@@ -25,8 +33,8 @@ def test_get_environments(self, client):
         assert len(environments) == 1
         assert environments[0]["name"] == env_obj.name
 
+    @pytest.mark.usefixtures("admin")
```

Comment:
why do you use fixtures here instead of the factory boy?

Reply:
Fixture uses FactoryBoy :)